Diseño paginas de crear propietario y login

santiago-cujaban opened the pull request.

Commit:
Adicion modulo admin y creacion de propietario

Colaboradores en este commit:
- Yilmar Alexis Malpica
- Juan Jose Sanchez Pineda
- Santiago Cujaban Neita

AGREGAR SI ES POSIBLE:
 - ! El login dentro de admin.component.html !
------------------------------------------------------------------

- Se añade la fuente de Krona One y un margen para el diseño general.  😄
- Falta agregar la conexion de crear propietario con el coreapiservice. 😖

NOTAS:
Si se presentan errores al generar un archivo desde la linea de comandos, descarguen nodejs 18.18.0

Co-Authored-By: JuanPi0205 <109747303+JuanPi0205@users.noreply.github.com>

Commit:
Mejora de pantalla crear propietario

Se ajusta para que se asemeje al diseño de Figma, el botón inicio sesion redirecciona a /admin. :smile:

Commit:
Adicion pagina de inicio de sesion

Colaboradores en este commit:

- Camilo Moreno
- Eduard Julian Ladino
- Jeisson Leonardo Buitrago

Co-Authored-By: Julian Ladino <118779478+Eduard-ladino@users.noreply.github.com>
Co-Authored-By: Leonardo110721 <150161452+Leonardo110721@users.noreply.github.com>

Camilo-Camargo commented on web/src/app/pages/admin/admin.component.css:
```diff
@@ -0,0 +1,81 @@
+display: flex;
+flex-direction: column;
+gap: 10px 0;
+}
+
+/* Estilo del div que contiene el checkbox y el label */
+.Terms{
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  align-items: center;
+}
+
+/* Estilo de la cajita del checkbox */
+.Terms input[type=checkbox]{
+  width: 20px;
+  height: 20px;
+}
+
+/* Estilo del label dentro de terminos */
+.Terms label{
+  font-size: 12px;
+}
+
+/* Estilo del div que contiene los botones del final */
+.Btns{
+  display: flex;
+  justify-content: space-between;
+  align-items: center; 
+}
+
+/* Estilo especifico del boton Registrarme */
+.Reg{
+  border: none;
+  color: #019D76;
+}
+
+/* Estilo especifico del boton Registrarme al pasar el mouse sobre este */
+.Reg:hover{
+  color: white;
```
podriamos eliminar comentarios, ya que el titulo de la clase es explicatoria.

santiago-cujaban replied:
Es posible, pero queremos mantener un poco de orden con estos.

Camilo-Camargo commented on web/src/app/pages/admin/admin.component.ts:
```diff
@@ -0,0 +1,12 @@
+})
+export class AdminComponent {
+  goToLogin(){
+    window.location.href="/admin"
```
no existe un routing de angular para hacer esto?

santiago-cujaban replied:
Si existe 😮 
import { Router } from '@angular/router';
se agrega en commits futuros. 👍

Commit:
Funcionalidad pantalla login

:cat:  Ahora en la pantalla login se puede enviar el json con la informacion correspondiente, siempre y cuando se seleccione el botón de terminos y condiciones. :cat:

Commit:
Funcionalidad pantalla crear propietario

:cat: Faltaria agregar verificaciones y el envio al servicio api.
Igualmente se mejora el enrutamiento. :cat:

santiago-cujaban reviewed (commented):
Se han revisado las inquietudes

Camilo-Camargo commented on web/src/app/pages/admin/create-owner/create-owner.component.ts:
```diff
@@ -0,0 +1,61 @@
+
+  // Información Formulario Vehiculo
+  vehicleForm = new FormGroup({
+    clase: new FormControl(''),
+    modelo: new FormControl(''),
+    importado: new FormControl(''),
+    decomisado: new FormControl(''),
+    marca: new FormControl(''),
+    carroceria: new FormControl(''),
+    blindado: new FormControl(''),
+    chatarrizado: new FormControl(''),
+    linea: new FormControl(''),
+    cilindraje: new FormControl(''),
+    caja_automatica: new FormControl(''),
+    estado: new FormControl(''),
+    val_antes_iva: new FormControl(''),
+    tonelaje: new FormControl(''),
+    carta_abiertas: new FormControl(''),
+    ext_dominio: new FormControl(''),
+    tipo_servicio: new FormControl(''),
+    pasajeros: new FormControl(0),
+    robado: new FormControl(''),
+    antiguo: new FormControl(''),
+    tipo_vehiculo: new FormControl(''),
+  })
```
Esto lo podríamos cambiar en inglés como esta en el modelo de base de datos, e igual con todas las variables, clases y funciones.

YilmarM replied:
Ya se modificaron las variables de Español a Ingles

LDRubiano committed:
Avaluos html

YilmarM committed:
Se cambiaron de "Español a Ingles " .

Se cambiaron el nombre de las variables según la estructura de la base de datos CODEZEN.

Commit:
Adicionado informacion pagina principal

:cat: Falta conectar la pantalla principal con el flujo de usuarios normales y el ingreso de admin :cat:

YilmarM committed:
Ventana consultar impuesto Vehiculo

Se agrega la ventana consultar vehiculo para que cuando el  usuario de clic en "Impuestos vehiculos" lo redirija a la ventana de consultar por "cedula y placa"

YilmarM committed:
Ventana Actualizar propietario

Se crea la ventana para actualizar el usuario por medio de la cedula. Y se crea  el enlace de la ventana de admin al hacer clic en actualizar

Co-Authored-By: Santiago-Cujaban <79294248+santiago-cujaban@users.noreply.github.com>

LDRubiano committed:
se sube pagina avaluos

JuanPi0205 committed:
pagina actualizar propietario con redireccionamiento del login

LDRubiano committed:
se cambian valores a ingles avaluos

LDRubiano committed:
se agrega css

LDRubiano committed:
se agregan los modulos y rutas

JuanPi0205 committed:
pagina consultar usuario hecha por eduard

LDRubiano committed:
se agregan clases para actualizar vehiculo

LDRubiano committed:
se arregla las clases de moduloa y rutas para avaluos

LDRubiano committed:
se arregla la clase para actualizar vehiculo

LDRubiano committed:
se agrega el css

LDRubiano committed:
se agregan los modulos y rutas de actualizar vehiculo

yosoyyomrj committed:
se Agrego clase actualizar vehiculo 2

yosoyyomrj committed:
Merge branch 'feature/appraisals_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/appraisals_ui

LDRubiano committed:
se arregla clases

LDRubiano committed:
se cambian variables

LDRubiano committed:
Merge branch 'feature/appraisals_ui' into feature/admin_ui

JuanPi0205 committed:
ventana pagos

Se agrega la ventana de pagos, segun el mockup

JuanPi0205 committed:
Merge branch 'feature/appraisals_ui' into feature/admin_ui

JuanPi0205 committed:
Ventana avaluos

Se agrega la ventana avaluos segun el mockup

Co-Authored-By: LDRubiano <133888277+LDRubiano@users.noreply.github.com>
Co-Authored-By: yosoyyomrj <130433171+yosoyyomrj@users.noreply.github.com>

LDRubiano committed:
Merge branch 'feature/admin_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/admin_ui

JuanPi0205 committed:
Correciones Pagina avaluos

Se hacen las respectivas correcciones visuales de la pagina avaluos

Co-Authored-By: LDRubiano <133888277+LDRubiano@users.noreply.github.com>
Co-Authored-By: YilmarM <142935863+YilmarM@users.noreply.github.com>
Co-Authored-By: Leonardo110721 <150161452+Leonardo110721@users.noreply.github.com>
Co-Authored-By: camigamer1234 <127879496+camigamer1234@users.noreply.github.com>

yosoyyomrj committed:
Merge branch 'feature/appraisals_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/appraisals_ui

LDRubiano committed:
Merge branch 'feature/appraisals_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/appraisals_ui

yosoyyomrj committed:
se agrego rutas y modulos para la clase actulizacion vehiculo 2

LDRubiano committed:
Merge branch 'feature/appraisals_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/appraisals_ui

yosoyyomrj committed:
se configura rutas y modulos de la clase actulizacion vehiculo 2

LDRubiano committed:
Merge branch 'feature/appraisals_ui' of https://github.com/Alejandro-Suarez-Rincon/impuestoVehicular into feature/appraisals_ui

LDRubiano committed:
se arreglan variables

LDRubiano committed:
se arreglan otras ariables

Camilo-Camargo committed:
Merge branch 'feature/appraisals_ui' into feature/admin_ui
